src/test/javascript/spec/component/about-us/about-us.component.spec.ts

```diff
@@ -47,4 +48,29 @@ describe('AboutUsComponent', () => {
             expect(getStaticJsonFromArtemisServerStub).toHaveBeenCalledTimes(1);
         });
     }));
+
+    it('load and display contributors', fakeAsync(() => {
+        const staticContentService = TestBed.inject(StaticContentService);
+        const profileService = TestBed.inject(ProfileService);
+
+        const fullName = 'Full Name';
+        const photoDirectory = 'Photo Directory';
+        const role = 'ADMIN';
+        const website = 'www.website.de';
+
+        const contributors = [new ContributorModel(fullName, photoDirectory, role, website)];
+
+        const getStaticJsonFromArtemisServerStub = jest.spyOn(staticContentService, 'getStaticJsonFromArtemisServer').mockReturnValue(of(new AboutUsModel([], contributors)));
+        const getProfileInfoSub = jest.spyOn(profileService, 'getProfileInfo');
+        getProfileInfoSub.mockReturnValue(
```

Comment:
I suggest to assert that this stub has been called in your ``then`` block below, like you did with the other stub.
Additionally, we can inline this (and change ``Sub`` to ``Stub``;D):
```suggestion
        const getProfileInfoStub = jest.spyOn(profileService, 'getProfileInfo').mockReturnValue(
```

Reply:
Unfortunately, prettier does not permit me to inline as you demonstrated on line 64, however, I still got rid of using the variable to call spyOn() anyways. Also, sub is now stub :)

![image](https://user-images.githubusercontent.com/67344158/155676866-200513d5-6e48-4f03-a64d-95feaaf6e69a.png)